services/ai/stt.ts

```
@@ -0,0 +1,242 @@
+    'nl-NL',
+    'sv-SE',
+    'da-DK',
+    'no-NO',
+    'fi-FI',
+  ];
+}
+
+/**
+ * Get supported languages for STT
+ */
+export function getSupportedLanguages(): string[] {
```

Biome (1.9.4)

[error] Shouldn't redeclare 'getSupportedLanguages'. Consider to delete it or rename it.

'getSupportedLanguages' is defined here:


(lint/suspicious/noRedeclare)